ios/Classes/FacebookModule.m

```diff
@@ -574,17 +580,25 @@ -(void)presentShareDialog:(id)args
     TiThreadPerformOnMainThread(^{
         FBSDKShareLinkContent *content = [[FBSDKShareLinkContent alloc] init];
         content.contentURL = [NSURL URLWithString:[params objectForKey:@"link"]];
-        content.contentDescription = [params objectForKey:@"description"];
-        if ([params objectForKey:@"name"] != nil) {
-            DEPRECATED_REPLACED_REMOVED(@"Facebook.presentShareDialog.name", @"5.0.0", @"5.0.0", @"Titanium.Facebook.presentShareDialog.title");
+        
+        NSString *description = [params objectForKey:@"description"];
+        NSString *title = [params objectForKey:@"title"];
+        NSString *picture = [params objectForKey:@"picture"];
+        
+        if (description != nil) {
+            NSLog(@"Ti.Facebook.presentShareDialog.description has been deprecated in Ti.Facebook 5.5.0 as of the Graph v2.9 changes.");
+            content.contentDescription = [params objectForKey:@"description"];
         }
-        if ([params objectForKey:@"title"] != nil){
-            content.contentTitle = [params objectForKey:@"title"];
+ 
+        if (title != nil) {
+            NSLog(@"Ti.Facebook.presentShareDialog.title has been deprecated in Ti.Facebook 5.5.0 as of the Graph v2.9 changes.");
+            content.contentTitle = title;
         }
-        if ([params objectForKey:@"caption"] != nil) {
-            DEPRECATED_REMOVED(@"Facebook.presentShareDialog.caption", @"5.0.0", @"5.0.0");
+        
+        if (picture != nil) {
+            NSLog(@"Ti.Facebook.presentShareDialog.picture has been deprecated in Ti.Facebook 5.5.0 as of the Graph v2.9 changes.");
+            content.imageURL = [NSURL URLWithString:picture];
```

Comment:
As per FBSDKShareLinkContent.h -
@property (nonatomic, readonly) NSURL *imageURL
DEPRECATED_MSG_ATTRIBUTE("imageURL is deprecated from Graph API 2.9");
imageURL has changed to read only, but you are assigning to it. It is giving build error. Please remove it.